database/author.js

```diff
@@ -1,9 +1,21 @@
 const mongoose = require('mongoose').set('debug', true);
-mongoose.connect('mongodb://localhost/author', {
-    useNewUrlParser: true,
-    useUnifiedTopology: true
-});
 
+async function resolveConnection() {
+    console.log('connecting');
+    try {
+        await mongoose.connect('mongodb://localhost:27017/author', {
+            family: 4,
```

Comment:
I never know Mongoose will try ipv6 first and then ipv4 if ipv6 fails. Per documentation by doing this will speed up the connection, nice work!

Reply:
Hi Ruoxi, thanks for the feedback. Yes, I did this while I was working on seeding data using MongoDb. I thought the connection time out was because of this, but it does not hurt to have it anyway even when we don't have problem connecting.